resources/github-comment-markdown.template

```diff
@@ -50,6 +50,16 @@
 | Total        | ${(testsSummary?.total) ?: 0}   |
 <%}%>
 
+#### Trends :test_tube:
+
+<% if(env?.JOB_URL != null && !env?.JOB_URL.equals('null')) {%>
+![Image of Build Times](${env?.JOB_URL}buildTimeGraph/png)
+<%}%>
+<% if(env?.BUILD_URL != null && !env?.BUILD_URL.equals('null')) {%>
+![Image of Tests](${env?.BUILD_URL}testReport/history/countGraph/png?start=0&amp;end=25)
+![Image of Cobertura](${env?.BUILD_URL}cobertura/graph)
```

Comment:
We might need to find a way to detect if it's there, otherwise there will be broken links :/ 

![image](https://user-images.githubusercontent.com/2871786/106656762-05fd0d00-6593-11eb-8af4-ddb9d270ff1e.png)

Reply:
by using the alt text I think is enough

`![not-available](foo.png)`

![not-available](foo.png)

Reply:
forget you have used it :D